app/controllers/DataUploadController.scala

```diff
@@ -77,37 +81,45 @@ class DataUploadController @Inject()(sessionService: SessionService,
     metrics.fileProcessingTimer(System.currentTimeMillis() - startTime, TimeUnit.MILLISECONDS)
 
   def processCsvFileDataFromFrontend(empRef:String): Action[JsValue] = authorisedActionWithBody(empRef) {
-    implicit request =>
+    implicit request: Request[JsValue] =>
+      Logger.warn("receiving request on V1 processCsvFileDataFromFrontend")
```

Comment:
For both processCSV methods, should this be Logger.warn or .debug? If the former, should we update it for processFile too?

Reply:
it should be debug indeed - updated